### PR TITLE
Image scan vulnerability metrics

### PR DESCRIPTION
**Pull request by IduVlad**
@c4po I've created a new pull request ( #27 can be deleted/declined) thanks

### CODE REVIEW

**Comment by c4po** [comment]
@IduVlad  can you create an issue for this PR. will be easy for me to maintain the change logs. Thanks

**Comment by c4po** [comment]
please also run a `make fmt` in your branch.

File: harbor_exporter.go [hunk 1]
```diff
@@ -126,6 +127,7 @@ func createMetrics(instanceName string) {
 	allMetrics["system_with_chartmuseum"] = newMetricInfo(instanceName, "system_with_chartmuseum", "If harbor has chartmuseum enabled", prometheus.GaugeValue, nil, nil)
 	allMetrics["system_notification_enable"] = newMetricInfo(instanceName, "system_notification_enable", "If notifications are enabled", prometheus.GaugeValue, nil, nil)
 	allMetrics["replication_latency"] = newMetricInfo(instanceName, "replication_latency", "Time in seconds to collect replication metrics", prometheus.GaugeValue, nil, nil)
+	allMetrics["image_vulnerability"] = newMetricInfo(instanceName, "image_vulnerability", "Get image vulnerability scan: None = 0, Low = 1, Medium = 2, High = 3 ", prometheus.GaugeValue, imageNames, nil)
```
Comment (dkulchinsky) [inline harbor_exporter.go]: small nit, but looks like missing `Critical = 4` in the description?

Comment (IduVlad) [reply harbor_exporter.go]: indeed, thanks for the hint. ```Critical = 4``` status is present now

File: metrics_repositories.go [hunk 2]
```diff
@@ -95,6 +103,52 @@ func (h *HarborExporter) collectRepositoriesMetric(ch chan<- prometheus.Metric)
...
+					tagName := tagData[key].Tags[0]
+
+					if len(scanData) < 16 {
+
+						continue
+					}
+					scan_overview := scanData["scan_overview"].(map[string]interface{})
+					severity := scan_overview["application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"].(map[string]interface{})
+
+					var vurnerability float64
```
Comment (dkulchinsky) [inline metrics_repositories.go:131]: s/vurnerability/vulnerability/

```suggestion
					var vulnerability float64
```

**Comment by c4po** [comment]
@IduVlad  I have concern on performance about to retrieve all artifacts information here. 
1. We should build a `collectArtifactsMetrics` function to get the artifact related metrics, not in the `collectRepositoriesMetric` function. so user can choose enable or disable this metric.
2. Artifacts API support pagination.
3. We can collect more metrics from artifacts with the API call,

**Pull request closed without merging**